Web/CSS2/styleTransform.css

```diff
@@ -0,0 +1,42 @@
+.rectangle{
+	width: 200px;
+	height: 200px;
+    margin:50px;
+	background-color:#5CADFF;
+    border: 3px solid navy;
+    
+}
+
+ 
+.rotate{
+	width: 200px;
+	height: 200px;
+    margin-left:350px;
+	margin-top:-250px;
+	background-color:#5CADFF;
+    border: 3px solid navy;
+    -webkit-transform: rotate(70deg); 
```

Comment:
We add -webkit-transform for cross browser support. Use this for skew and scale too.
And this will not work in IE9. Add style to support in this browser too.